### PR TITLE
LSC wireless EU support + Reduce lag a bit

### PR DESCRIPTION
**Pull request by Connor-Colenso**
Screenshot 2022-08-13 at 17 25 45

This multi was also recalculating expensive bigintegers quite a lot, tried to minimise that.

### CODE REVIEW

File: src/main/java/common/tileentities/GTMTE_LapotronicSuperCapacitor.java [hunk 1]
```diff
@@ -508,7 +599,13 @@ public String[] getInfoData() {
 		ll.add("Maintenance Status: " + ((super.getRepairStatus() == super.getIdealStatus())
 				? EnumChatFormatting.GREEN + "Working perfectly" + EnumChatFormatting.RESET
 				: EnumChatFormatting.RED + "Has Problems" + EnumChatFormatting.RESET));
-		ll.add("---------------------------------------------");
+        ll.add("Wireless mode: " + (wireless_mode
+            ? EnumChatFormatting.GREEN + "enabled" + EnumChatFormatting.RESET
+            : EnumChatFormatting.RED + "disabled" + EnumChatFormatting.RESET));
+        ll.add(GT_Values.TIER_COLORS[9] + GT_Values.VN[9] + EnumChatFormatting.RESET + " Capacitors detected: " + uhv_cap_count);
+        ll.add("Total wireless EU: " + EnumChatFormatting.RED + GT_Utility.formatNumbers(getUserEU(global_energy_user_uuid)));
+        ll.add("Wireless Network EU/t: " + EnumChatFormatting.RED + GT_Utility.formatNumbers(getUserEU(global_energy_user_uuid)));
```
Comment (chochem) [inline src/main/java/common/tileentities/GTMTE_LapotronicSuperCapacitor.java]: this shouldnt be the same value as the line above it

**Comment by chochem** [comment]
any idea why/if this broke the Average EU IN/OUT scan/display? see https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/11056

**Comment by Connor-Colenso** [comment]
> any idea why/if this broke the Average EU IN/OUT scan/display? see [GTNewHorizons/GT-New-Horizons-Modpack#11056](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/11056)

Fixed

**Comment by Connor-Colenso** [comment]
Intellij not working rn fixed in dev

**Review by Dream-Master — APPROVED** [review]
It works and it is live on zeta

**Pull request merged**